scripts/insert_into_appointments.sql

```diff
@@ -1 +1,27 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(date, time, comment)
+  SELECT 'August 12, 2016',
+    '10am',
+    'Marsha Wilcox is seeing Dr. Hilton',
+    (SELECT id FROM doctors WHERE given_name = 'Vickie' AND surname = 'Hilton'),
+    (SELECT id FROM patients WHERE given_name = 'Marsha' AND surname = 'Wilcox')
+  UNION
+  SELECT 'August 17, 2016',
+    '1pm',
+    'Ivory Talley is seeing Dr. Spencer',
+    (SELECT id FROM doctors WHERE given_name = 'Spencer' AND surname = 'Hubbard'),
+    (SELECT id FROM patients WHERE given_name = 'Ivory' AND surname = 'Talley')
+  UNION
+  SELECT 'August 12, 2016',
+    '10am',
+    'Marsha Wilcox is seeing Dr. Spencer',
+    (SELECT id FROM doctors WHERE given_name = 'Spencer' AND surname = 'Hubbard'),
+    (SELECT id FROM patients WHERE given_name = 'Marsha' AND surname = 'Wilcox')
+  UNION
+  SELECT 'August 17, 2016',
+    '1pm',
+    'Blake Hodge is seeing Dr. Hilton',
+    (SELECT id FROM doctors WHERE given_name = 'Hilton' AND surname = 'Vickie'),
+    (SELECT id FROM patients WHERE given_name = 'Blake' AND surname = 'Hodge')
+;
```

Comment:
I see what you did when writing the comment in and the full date. Something like this next time 

```
INSERT INTO appointments(date, time, doctor_id, patient_id)
  SELECT '20150812', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hilton'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Talley')
  UNION
  SELECT '20150817', '1000',
    (SELECT id FROM doctors WHERE surname = 'Hubbard'),
    (SELECT id FROM patients WHERE surname = 'Wilcox')
  UNION
  SELECT '20150817', '1300',
   (SELECT id FROM doctors WHERE surname = 'Hilton'),
   (SELECT id FROM patients WHERE surname = 'Hodge')
;
```